Lesson21/Mesut/app.js

```diff
@@ -1,29 +1,31 @@
-// Events to check that page is loaded (should be applıed to window object):
+// Events to check that page is loaded (should be applied to window object):
 // DOMContentLoaded - DOM fully loaded
 // load - all elements (including images and styles) are loaded
-window.addEventListener('load',function(){
+window.addEventListener("load", function () {
   // Select all images
-  const images= document.querySelectorAll("img"); // Array of image
+  const images = [...document.querySelectorAll("img")]; // Array of image
```

Comment:
`document.querySelectorAll` will return a collection of nodes which is not exactly a JS array so you won't be able to use some of array method on it. However, here you use `forEach` which is a method that can be used on collection of nodes so you don't have to convert it to array.